### PR TITLE
Implemented feedback on PR review #252

### PR DESCRIPTION
**Pull request by sandervanhooft**

### CODE REVIEW

**Comment by sandervanhooft** [comment]
#247

**Comment by sandervanhooft** [comment]
#252

File: src/Resources/OrderLine.php [hunk 1]
```diff
@@ -274,11 +274,25 @@ public function isSurcharge()
...
-    public function cancel()
+    /**
+     * Cancel this order line.
+     *
+     * @param  array|null $data
+     * @return null
+     */
+    public function cancel($data = [])
     {
+        $resource = "orders/" . urlencode($this->orderId) . "/lines/" . urlencode($this->id);
```
Comment (Smitsel) [inline src/Resources/OrderLine.php]: There is a _self link for this present on the `OrderLine`

Comment (Smitsel) [reply src/Resources/OrderLine.php]: Never mind. This is not present.

**Pull request merged**